code/modules/heavy_vehicle/mech_damage.dm

```diff
@@ -99,4 +113,21 @@
 		if(!hatch_closed || !prob(body.pilot_coverage))
 			for(var/thing in pilots)
 				var/mob/pilot = thing
-				pilot.emp_act(severity)
\ No newline at end of file
+				pilot.emp_act(severity)
+
+/mob/living/heavy_vehicle/fall_impact(levels_fallen, stopped_early = FALSE, var/damage_mod = 1)
```

Comment:
I don't know if this is right place to put it. It seems like everything related to Z level is inside of movement.dm. what so you think Skull, Matt?